app/code/Magento/Store/Console/Command/StoreListCommand.php

```diff
@@ -0,0 +1,57 @@
+<?php
+namespace Magento\Store\Console\Command;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Command\Command;
+
+/**
+ * Class StoreListCommand
+ * @package Magento\Store\Console\Command
+ */
+class StoreListCommand extends Command
```

Comment:
Have you tried to run these commands when magento is not installed? Please check this case.  Requesting StoreManager in constructor might lead to DB calls.